validate.go

```diff
@@ -394,7 +422,7 @@ func IsAbsolutePathValidation(v *validator.Validate, topStruct reflect.Value, cu
 		return true
 	}
 
-	return strings.HasPrefix(field.String(), "/") || strings.HasPrefix(field.String(), "{{repl")
+	return strings.HasPrefix(field.String(), "/") || strings.HasPrefix(field.String(), "{{repl ")
```

Comment:
Could `{{repl` be followed by a tab?  I'd just leave the space out.  It will fail elsewhere if their templating syntax is not correct.